tools/devops/automation/scripts/GitHub.psm1

```diff
@@ -268,6 +298,138 @@ class GitHubComments {
 
         return $this.NewComment($msg)
     }
+
+    [object] GetCommentsForPR ($prId) {
+        # build the query, create the json and perform a rest request againt the grapichQl api
+        $headers = @{
+            Authorization = ("Bearer {0}" -f $this.Token)
+        }
+
+        $query = @"
+query {
+    repository(owner:"$($this.Org)", name:"$($this.Repo)"){
+        pullRequest(number: $prID) {
+            comments(first: 100) {
```

Comment:
Is the 100 a limit on the number of comments to grab? e.g. "100 comments is enough for anyone"

Reply:
Is the limit of the API. I'd be surprise we have more than 100 comments per PR. We can do pagination if needed. But Is extra logic that probably is not needed.